frontend/templates/process.html

```diff
@@ -25,26 +25,38 @@
     <div class="process-form-block">
 
         <form id="process-form" name="process-form" data-name="process-form" class="process-form">
+                <div class="description-columns">
+                    <div class="div-wrapper">
+                        <div class="left-div">
+                            
+                            <label for="process-name-textarea"
+                                    class="entry-label">Name des Prozesses
+                            </label>
+                            <input  type="text"
+                                    name="process-name-textarea"
+                                    maxlength="300"
+                                    id="process-name-textarea"
+                                    class="textfield">
 
-            <div class="description-columns">
-                <div class="div-wrapper">
-                    <div class="left-div">
-                        <label for="process-name-textarea"
-                               class="entry-label">Name des Prozesses
-                        </label>
-                        <input type="text"
-                               name="process-name-textarea"
-                               maxlength="300"
-                               id="process-name-textarea"
-                               class="textfield">
-                        <label for="process-beschreibung-textarea"
-                               class="entry-label">Beschreibung des Prozesses
-                        </label>
+                            <label for="process-responsible-person-textarea"
+                                    class="entry-label">Responsible Person
```

Comment:
```suggestion
                            <label for="process-responsible-person-textarea"
                                    class="entry-label" data-i18n="responsible-person">
```

Müsste das nicht so funktionieren, wenn "responsible-person" einfach schnell im en.json als eine Zeile hinzugefügt wird?